src/main/java/frc/robot/Constants.java

```diff
@@ -39,4 +39,23 @@ public static class WheelConstants {
     
 
   }
+  //All measurements in inches
+  public static class ArenaConstants {
+    public static final double midCubeHeight = 18.25;
+    public static final double highCubeHeight = 35.5;
+    public static final double midCubeFromFront = 14.25;
+    public static final double highCubeFromFront = 31.625;
+
+    public static final double kmidConeHeightMeters = Units.inchesToMeters(34);
+    public static final double khighConeHeightMeters = Units.inchesToMeters(46);
+    public static final double kmidConeFromFrontMeters = Units.inchesToMeters(22.75);  //how far the arm will have to reach to get to mid peg
+    public static final double highConeFromFront = 37.75;
+
```

Comment:
Was unable to validate/figure out where this height came from. Assume the units is in inches.
Rename to follow naming convention